src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -168,14 +168,15 @@ def save_local_context(self, parsed_args, argument_definitions, specified_argume
 
         # print warning if there are values saved to local context
         if local_context_args:
-            logger.warning('Local context is turned on. Its information is saved in working directory %s. You can '
-                           'run `az local-context off` to turn it off.',
+            logger.warning('Parameter persistence is turned on. Its information is saved in working directory %s. '
+                           'You can run `az config param-persist off` to turn it off.',
                            self.local_context.effective_working_directory())
             args_str = []
             for name, value in local_context_args:
                 args_str.append('{}: {}'.format(name, value))
-            logger.warning('Your preference of %s now saved to local context. To learn more, type in `az '
-                           'local-context --help`', ', '.join(args_str) + ' is' if len(args_str) == 1 else ' are')
+            logger.warning('Your preference of %s now saved to parameter persistence. To learn more, type in `az '
+                           'config param-persist --help`',
+                           ', '.join(args_str) + (' is' if len(args_str) == 1 else ' are'))
```

Comment:
Add `()` to fix the bug that the message is showing `' are'` instead of `', '.join(args_str) + ' are'` when `len(args_str) > 1`.